miden/src/cli/verify.rs

```diff
@@ -17,33 +18,39 @@ pub struct VerifyCmd {
     output_file: Option<PathBuf>,
     /// Path to proof file
     #[clap(short = 'p', long = "proof", value_parser)]
-    proof_file: PathBuf,
+    proof_file: Option<PathBuf>,
     /// Program hash (hex)
-    #[clap(short = 'h', long = "program-hash")]
+    #[clap(short = 'x', long = "hash")]
     program_hash: String,
```

Comment:
I would probably keep the long name as `program-hash` as it is more self-describing.